src/components/Header/style.scss

```diff
@@ -1,11 +1,14 @@
 .c-header {
```

Comment:
Could you use variables to define margins and box sizes please? It should be defined at top of style document.